packages/web/components/stake/unbonding-in-progress.tsx

```diff
@@ -0,0 +1,78 @@
+import { CoinPretty } from "@keplr-wallet/unit";
+import React from "react";
+import { useTranslation } from "react-multi-lang";
+
+import { useStore } from "~/stores";
+
+export const UnbondingInProgress: React.FC<{
+  unbondings: {
+    completionTime: string;
+    balance: CoinPretty;
+  }[];
+}> = ({ unbondings }) => {
+  const t = useTranslation();
+  const { priceStore } = useStore();
+
+  function formatUnbondings(
```

Comment:
Is this function definition within the render body needed? It's somewhat hard to follow the logic, given the invocation in the render body below.

Reply:
given that this is just formatting for this view, it didn't seem like it was worth it to extract this function. Since this view is only rendered once, any performance difference is negligible.